src/data_cleaning.R

```diff
@@ -0,0 +1,54 @@
+#preamble
+rm(list=ls()) # clear vars
+cat("\014") # clear console
+library(tidyverse)
+library(lubridate)
+library(ggplot2)
+library(tsibble)
+
+
+setwd("C:\\Users\\Yamni\\Desktop\\TNMD-mini-hackathon\\air_quality_data")
```

Comment:
If you're not familiar with the [here package](https://here.r-lib.org/), its a great alternative for dealing with paths in project oriented workflows (allows you to essentially provide a relative path given the project directory, instead of the absolute path).

Another option for loading in the data here would be to download it directly from GitHub in the script.